jest.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Nit: Should we put these Jest config files in `tools/jest` to keep the root clean'ish?

Reply:
Sure, I didn't think we had a cleanish root, but I guess all those dotfiles are normally hidden 😛